docs/CUSTOMIZING.rst

```diff
@@ -128,16 +281,33 @@ Using a Schema with no ``InputFileName`` Field
 If you would like to use Optical Character Recognition (OCR) to match images in your import directory with rows on your upload excel/csv file, you need to make the following changes to your schema:
 
 * Ensure your schema does not have a field ``InputFileName``, and there is no corresponding column on your upload file
-* Ensure your schema contains a column for target text, and that the column is specified in ``girder.local.conf``. The property to set is ``import_text_association_column``.
+* Ensure your schema contains one or more columns for target text, and that the column is specified in ``girder.local.conf``. The property to set is ``import_text_association_columns``.
 
 The target text column should contain label text of WSIs in the import directory. During the ingest process, all images in your specified import directory will be ingested into the ``Unfiled`` folder in the ``WSI DeID`` collection. Then, those images will be associated with data found on the upload file. Progress can be tracked as a girder job. If no match can be determined, images will remain in the ``Unfiled`` folder. Images with a match will be transferred to the ``AvailableToProcess`` folder.
 
 If ``InputFileName`` is added to the list of export fields in the ``upload_metadata_for_export_report`` settings, then the original file name will be included in the export report.
 
+.. code-block:: python
+
+  [wsi_deid]
+  ...
+  # Attempt to match OCR results with data in these columns from the import excel file
+  import_text_association_columns = ["SurgPathNum", "First_Name", "Last_Name", "Date_of_Birth"]
+  ...
+
+
 Choosing Custom Metadata To Add to Exported Images
 ++++++++++++++++++++++++++++++++++++++++++++++++++
 
-By default, any metadata from the upload excel/csv file except ``InputFileName`` is added to the exported files in the ``ImageDescription`` or ``Software`` metadata tags (depending on format) with a prefix of ``CustomField.`` (e.g., ``CustomField.Proc_Type`` will be added with data from a ``Proc_Type`` column in the ypload file).  This can be configured with the ``upload_metadata_add_to_images`` setting in ``girder.local.conf``.  If present and not None, this setting is a list of columns with will be added to the exported file.
+By default, any metadata from the upload excel/csv file except ``InputFileName`` is added to the exported files in the ``ImageDescription`` or ``Software`` metadata tags (depending on format) with a prefix of ``CustomField.`` (e.g., ``CustomField.Proc_Type`` will be added with data from a ``Proc_Type`` column in the upload file).  This can be configured with the ``upload_metadata_add_to_images`` setting in ``girder.local.conf``.  If present and not None, this setting is a list of columns with will be added to the exported file.
+
+.. code-block:: python
+
+  [wsi_deid]
+  ...
+  upload_metadata_add_to_images = ["Proc_Seq", "Proc_Type", "Spec_Site"]
```

Comment:
The fragment with comments and this value shown as either `None` or a list of values should be moved from the export fields section to here.

Reply:
b688d01